event_sink_clickhouse/sinks/base_sink.py

```diff
@@ -351,3 +366,14 @@ def is_enabled(cls):
         )
 
         return enabled or waffle_flag.is_enabled()
+
+    @classmethod
+    def get_sink_by_model(cls, model):
```

Comment:
Can you rename this to `get_sink_by_model_name`? I was confused thinking it was a model instance.